### PR TITLE
ctxless cli node utils

### PR DESCRIPTION
**Pull request by tzemanovic**
## Describe your changes

Do not load `Context` for node utils (`test-genesis`) cmds

## Checklist before merging 
- [ ] If this PR has some consensus breaking changes, I added the corresponding `breaking::` labels
    - This will require 2 reviewers to approve the changes

### CODE REVIEW

**Comment by tzemanovic** [comment]
I'm having some issues with the `test-genesis` cmd - it crashes on spawning the dummy task:

```
The application panicked (crashed).
Message:  there is no reactor running, must be called from the context of a Tokio 1.x runtime
Location: crates/node/src/lib.rs:983
```

**Comment by tzemanovic** [comment]
> I'm having some issues with the `test-genesis` cmd - it crashes on spawning the dummy task:
> 
> ```
> The application panicked (crashed).
> Message:  there is no reactor running, must be called from the context of a Tokio 1.x runtime
> Location: crates/node/src/lib.rs:983
> ```

I fixed it by removing the dummy task and also added an e2e test to cover this

File: crates/tests/src/e2e/ledger_tests.rs [hunk 1]
```diff
@@ -2556,3 +2557,32 @@ fn masp_txs_and_queries() -> Result<()> {
 
     Ok(())
 }
+
+/// Test localnet genesis files with `namada node utils test-genesis` command.
+#[test]
+fn test_localnet_genesis() -> Result<()> {
+    let loc = format!("{}:{}", std::file!(), std::line!());
```
Comment (sug0) [inline crates/tests/src/e2e/ledger_tests.rs:2564]: can't this be implemented as an integration test?

Comment (tzemanovic) [reply crates/tests/src/e2e/ledger_tests.rs:2564]: we don't have node cmds in integration tests :/

**Review by sug0 — APPROVED** [review]
LGTM

**Review by batconjurer — APPROVED** [review]

**Pull request merged**